lib/engine/game/g_18_eus/round/stock.rb

```diff
@@ -1,77 +1,21 @@
 # frozen_string_literal: true
 
 require_relative '../../../round/stock'
+require_relative 'bidbox_auction'
 
 module Engine
   module Game
     module G18EUS
       module Round
         class Stock < Engine::Round::Stock
+          include Engine::Game::G18EUS::Round::BidboxAuction
           attr_accessor :bids, :taken_loans, :paid_loans
```

Comment:
bids not defined here. Also, the other two should be moved to where they are used.

Reply:
Moved